### PR TITLE
ADAPT-5863 | Update to Source Sans 3 and Source Serif 4; minor updates

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Update font stack and import to Source Sans 3 and Source Serif 4, with the old versions as fallback (in the mixin and classes)
- Remove ie11 from browerslistrc
- linter fix

# Needed By (Date)
- Soonish

# Urgency
- 3

# Steps to Test

1. Look at code
2. The preview is not really tied to this PR unfortunately, but you could pull this branch down locally, and do `npm run dev`, `npm run start`, and inspect the text like I did here. It should tell you the rendered fonts are Source Sans 3
Screenshot 2023-07-25 at 2 57 12 PM

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- ADAPT-5795, https://stanfordits.atlassian.net/browse/ADAPT-5863
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@sherakama Not sure if there's an ADAPT ticket related to the Source Sans and Serif font update. If there is, please LMK 😄 I want to update it in Decanter first, then I can use it in Giving.

File: core/src/js/components/accordion/accordion.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ Array.prototype.forEach.call(titleButtons, btn => {
     if (!isExpanded(btn)) {
       setExpanded(btn, 'true');
       setHidden(btn.parentNode.nextElementSibling, 'false');
-    } else {
+    }
```
Comment (yvonnetangsu) [inline core/src/js/components/accordion/accordion.js:26]: This is just a linter complain

File: core/src/scss/core/fonts/_fonts.scss [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
-@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
+// Sans font: Source Sans 3.
+@warn "Sans serif font weight 200, 300, and 900 will be removed in Decanter Version 7.";
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+3:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
 
-// Serif font: Source Serif Pro.
-@import url('https://fonts.googleapis.com/css2?family=Source+Serif+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
+// Serif font: Source Serif 4.
+@warn "Serif font weight 200, 300, and 900 will be removed in Decanter Version 7.";
+@import url('https://fonts.googleapis.com/css2?family=Source+Serif+4:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
```
Comment (yvonnetangsu) [inline core/src/scss/core/fonts/_fonts.scss:17]: Update the font imports here. Also added a warning that we're removing 200, 300, and 900 font weight in Decanter 7. Most people don't use 900 in their projects, 200 and 300 are not accessible 🤷🏼‍♀️ If they somehow really need it, they can always add it back into their own projects

File: core/src/scss/utilities/mixins/icons-logos/_logo.scss [hunk 3]
```diff
@@ -8,7 +8,7 @@
 /// @group brand
 @mixin logo {
   display: inline-block;
-  font-family: Stanford, 'Source Serif Pro', Georgia, Times, 'Times New Roman', serif;
+  font-family: Stanford, 'Source Serif 4', 'Source Serif Pro', Georgia, Times, 'Times New Roman', serif;
```
Comment (yvonnetangsu) [inline core/src/scss/utilities/mixins/icons-logos/_logo.scss:11]: Add Source Serif 4 in the logo mixin

File: core/src/scss/utilities/variables/core/_typography.scss [hunk 4]
```diff
@@ -60,15 +60,15 @@ $su-intro-line-height: 1.5 !default;
...
+$su-font-sans: 'Source Sans 3', 'Source Sans Pro', $font-stack-helvetica !default;
 
 ///
 /// font-serif font family.
 ///
 /// @name $su-font-serif
 ///
 /// @group variable
-$su-font-serif: 'Source Serif Pro', $font-stack-georgia !default;
+$su-font-serif: 'Source Serif 4', 'Source Serif Pro', $font-stack-georgia !default;
```
Comment (yvonnetangsu) [inline core/src/scss/utilities/variables/core/_typography.scss:71]: The 2 places where we update the font stack is in the typography core variables. These are then referenced in the mixins and classes

File: core/src/templates/core/fonts/fonts.twig [hunk 5]
```diff
@@ -1,7 +1,6 @@
 <div class="su-sans">
-  <h2>Source Sans Pro</h2>
-  <p class="su-intro-text">Source Sans Pro is our primary sans-serif font. Source Sans Pro should be used in unit signatures alongside the Stanford wordmark and should be used as the primary sans-serif font in your print, web and social media communications. For
-    situations where Source Sans cannot be used, use News Gothic or Gill Sans as fallback fonts.</p>
+  <h2>Source Sans 3</h2>
```
Comment (yvonnetangsu) [inline core/src/templates/core/fonts/fonts.twig:2]: These are just in the website documentation

File: .browserslistrc [hunk 6]
```diff
@@ -1,2 +1 @@
 last 2 versions
```
Comment (sherakama) [inline .browserslistrc:1]: Might want to change this while we are here to `defaults`

https://github.com/browserslist/browserslist#best-practices

File: core/src/scss/core/fonts/_fonts.scss (hunk 2 above)
Comment (sherakama) [reply core/src/scss/core/fonts/_fonts.scss:17]: Cool.

File: .browserslistrc (hunk 6 above)
Comment (yvonnetangsu) [reply .browserslistrc:1]: Sounds good to me 👍🏼

**Comment by sherakama** [comment]
> @sherakama Not sure if there's an ADAPT ticket related to the Source Sans and Serif font update. If there is, please LMK 😄 I want to update it in Decanter v6 first, then I can use it in Giving.

I thought we added something... Couldn't find it.

Here you go: https://stanfordits.atlassian.net/browse/ADAPT-5863

**Review by sherakama — APPROVED** [review]
This seems fine to me but I want @jenbreese to give the official 👍

**Comment by yvonnetangsu** [comment]
@sherakama This is strange, I just pushed up a new commit with the browserslist update to `defaults` with a few dist file changes due to running `npm run publish`, but I'm not seeing the new commit here 😬 GitHub status seems ok.

**Comment by yvonnetangsu** [comment]
Screenshot 2023-07-25 at 4 25 30 PM
Screenshot from my SourceTree 👀

**Comment by yvonnetangsu** [comment]
@sherakama This is so strange, the update browerslistrc commit was lost on this GitHub page, but my jiggle commit is showing up, and now it's showing the previously invisible commit with the browerslistrc update now 😬

File: core/src/scss/core/fonts/_fonts.scss (hunk 2 above)
Comment (jenbreese) [reply core/src/scss/core/fonts/_fonts.scss:17]: @yvonnetangsu We are using the 900 font weight on our ChEM-H subtheme project.

**Comment by yvonnetangsu** [comment]
Merging as patch since user doesn't to do anything other than updating the Decanter version.

**Pull request merged**